src/content/tools/doc-comment-references.md

```diff
@@ -0,0 +1,125 @@
+
+These doc comments contain references to the `String` class, the `object`
+parameter, and the `Future.value` constructor.
+
+## Features of references
+
+There are several benefits to referring to code elements with doc comment
+references:
+
+### Editor features
+
+Doc comment references enable several IDE features:
+
+- **Code completion**
+  An element's name can be code-completed within square brackets.
+- **Rename refactoring**
+  When an element is renamed via an IDE command, it can rewrite uses of that
+  element, including references in doc comments.
+- **Find references**
+  When an IDE lists all "references" to an element, it can include references in
+  doc comments.
+- **Go to definition**
+  An IDE can also provide Go-to-definition support at the location of a doc
+  comment reference.
+
+:::tip
+The [comment_references](https://dart.dev/lints/comment_references) lint rule
+can help to ensure that doc comment references are valid, avoiding typos and
+mis-uses. Keeping doc comment references valid ensures that these IDE features
+are enabled for each reference.
+:::
+
+### API documentation
+
+In API documentation generated by [`dart doc`](/tools/dart-doc), a doc comment reference is
+linked, if possible, to the documentation page of the element being referenced.
+If the element does not have a documentation page (for example, a function
+parameter, a type variable, or a private class), then no link is created.
+
+## What can be referenced
+
+Here is a quick reference of what can be referenced, and how:
+
+<table>
+  <tr><th>Member</th><th>How to reference</th></tr>
+  <tr>
+    <td>
+Library members (classes, constants, enums, named extensions, extension types, functions, mixins, type aliases)
+    </td>
+    <td>
+Reference with their name, if they are in-scope, prefixed with an import prefix if appropriate.
+
+```dart
+/// [List] is in scope.
+/// So is [math.max].
+```
+    </td>
+  </tr>
+</table>
+
+Most library members can be referenced in a doc comment, including classes,
+constants, enums, named extensions, extension types, functions, mixins, and
+type aliases. This includes all in-scope library members, either declared
+locally, or imported. Library members that are imported with an import prefix
+can be referenced with the prefix. For example:
+
+```dart
+import 'dart:math' as math;
+
+/// [List] is in scope.
+/// So is [math.max].
+int x = 7;
+```
+
+Most members of a class, an enum, an extension, an extension type, and a mixin
+can also be referenced. A reference to a member that is not in scope must be
+qualified (prefixed) with its container's name. For example the `wait` static
+method on the `Future` class can be referenced in a doc comment with
+`[Future.wait]`. This is true for instance members as well; the `add` method
+and the `length` property on the `List` class can be referenced with
+`[List.add]` and `[List.length]`. In-scope members like these can even be
+referenced without the qualifying container name:
```

Comment:
"In-scope members like these" makes it sound like the qualified examples in the previous sentence are "in-scope". Perhaps clarify with a conditional clause, something like:

```
When container members are in-scope, such as in an instance method's doc comment, they can be referenced without the qualifying container name:
```

Reply:
I like it. Done.